atlp-rwanda/cabal-phantom-frontend
# #klVOKtOs Simulate bus motion

#### What does this PR do?
 Requests to merge ft-frontend-simulate-bus-motion-klVOKtOs into develop

#### Description of Task to be completed?

- Add a page with a map
- The driver should be able to command the bus to start, move and stop.
- The driver should be able to command the bus to speed up and slow down.
- The driver should be able to update bus info on bus stops. e.g. number of passengers alighted and boarded.
- The **users** should be able to view the bus movement in real-time.
- The page is responsive.
- Skeleton UIs/screen are/is shown when the page loads/is loading.
- Appropriate error messages are displayed on unsuccessful attempts.

#### How should this be manually tested?

- clone this repo and do git checkout ft-frontend-simulate-bus-motion-klVOKtOs
- Open phantom app in you browser
- Input you origin and destination then click Search button
- check your location on the map

#### What are the relevant pivotal tracker stories?
[e8DmIOfi
](https://trello.com/c/e8DmIOfi/14-simulate-the-bus-motion)

#### Screenshots (if appropriate)
<img width="1440" alt="Screen Shot 2021-01-07 at 12 47 54" src="https://user-images.githubusercontent.com/68019992/103900337-5900b300-5100-11eb-9c1c-44171a0056ad.png">
